src/map/script.c

```diff
@@ -9013,6 +9013,32 @@ BUILDIN(getgmlevel)
 	return true;
 }
 
+/// set the group ID of the player.
+/// setgroupid( <new group id>,{<account ID> or <Player Name>} )
+/// return 1 on success, 0 if failed.
+BUILDIN(setgroupid) {
+	TBL_PC* sd;
```

Comment:
possible to use doesn't mean we should use it, Hercules is under Unix Kernel syntax write rules, which doesn't allow this :)